pages/board.js

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import Profiles from "../components/Board/Profiles";
+
+const Board = () => {
+	return (
+		<div>
```

Comment:
If you are only returning a single item, you can remove the `<div>` tag, sometimes it causes CSS errors since the `<Profiles/>` component has a parent component which may not be how the CSS was supposed to be